main.go

```diff
@@ -59,26 +64,41 @@ func main() {
 	}
 
 	config = Configuration{
-		AccessKey: os.Getenv("AWS_ACCESS_KEY_ID"),
-		SecretKey: os.Getenv("AWS_SECRET_ACCESS_KEY"),
-		Bucket: os.Getenv("AWS_BUCKET"),
-		Region: os.Getenv("AWS_REGION"),
-		Database: os.Getenv("DATABASE_URL"),
-		Port: port,
+		AccessKey:    os.Getenv("AWS_ACCESS_KEY_ID"),
+		SecretKey:    os.Getenv("AWS_SECRET_ACCESS_KEY"),
+		Bucket:       os.Getenv("AWS_BUCKET"),
+		Region:       os.Getenv("AWS_REGION"),
+		Database:     os.Getenv("DATABASE_URL"),
+		Port:         port,
+		RollbarToken: os.Getenv("ROLLBAR_KEY"),
+		Environment:  getenvOrDefault("APP_ENV", "development"),
+		GitCommit:    os.Getenv("SOURCE_VERSION"),
+		GithubRepo:   "github.com/CatchRelease/s3zipper",
 	}
 
-	fmt.Println("AWS ACCESS KEY", config.AccessKey)
-	fmt.Println("AWS SECRET KEY", config.SecretKey)
+	fmt.Println("ENVIRONMENT", config.Environment)
+	fmt.Println("AWS ACCESS KEY ID", config.AccessKey)
+	fmt.Println("AWS SECRET ACCESS KEY", config.SecretKey)
 	fmt.Println("AWS REGION", config.Region)
 	fmt.Println("AWS BUCKET", config.Bucket)
 	fmt.Println("DATABASE_URL", config.Database)
 
-	initAwsBucket()
-	initDB()
+	initRollbar()
 
-	fmt.Println("Running on port", config.Port)
-	http.HandleFunc("/", handler)
-	http.ListenAndServe(":"+strconv.Itoa(config.Port), nil)
+	var finishSetupAndListen = func() {
```

Comment:
splitting this into a fully separate function seemed unnecessary since it's kind of a hodgepodge